src/libANGLE/renderer/gen_angle_format_table.py

```diff
@@ -239,6 +239,8 @@ def get_component_alignment_mask(channels, bits):
 
 def get_vertex_attrib_type(format_id):
 
+    if "R32G8X24_TYPELESS" in format_id:
+        return "Float"
```

Comment:
Can't we let this run into `"InvalidEnum"`? Also curious why is this a `"Float"`, none of the D32S8 formats are allowed for vertex input - in D3D11 at least, do you have a reference for OpenGL?

Reply:
I deleted this piece of code to handle `R32G8X24_TYPELESS` similar as other `_TYPELESS` formats: It seems the last member of the generated structure describes the underlying type that determines the alignment, which is UnsignedInt in case of R32 and Byte in case of R8 etc...

Reply:
As discussed: all typeless should probably be `InvalidEnum` but we're fine with not fixing it (we aim for a smaller diff), since it has no runtime consequence for those types anyway.